app/api/api_v1/endpoints/auth.py

```
@@ -127,21 +79,32 @@ async def google_auth(
-    )
+    auth_service = AuthService(session)
+    auth_service.can_complete_profile(current_user, request)
+    return await auth_service.complete_profile(current_user, request)
+
+
+@router.post("/update-phone-number", response_model=UserOut)
+def update_phone_number(
+    request: UpdatePhoneRequest,
+    current_user: VerifiedUserDep,
+    session: SessionDep,
+    api_key: str = Security(api_key_header),
```

Ruff (0.12.2)

Unused function argument: `api_key`

(ARG001)